common/src/main/kotlin/io/homeassistant/companion/android/common/data/websocket/impl/WebSocketCoreImpl.kt

```diff
@@ -155,9 +158,14 @@ internal class WebSocketCoreImpl(
                                 ),
                             )
                             Timber.d("Sending message ${supportedFeaturesMessage["id"]}: $supportedFeaturesMessage")
-                            it.send(
+                            val result = it.send(
                                 webSocketMapper.writeValueAsString(supportedFeaturesMessage),
                             )
+                            if (!result) {
+                                // Something got wrong when sending the message but we should not change the status of the
+                                // connection here. If an error occur in the WS it will be handled in the onFailure.
+                                Timber.e("Unable to send supported features message")
```

Comment:
As it is not fatal, I would downgrade this to a warning. In case of a major error `onFailure` will log after this one with the error level.